python/tests/lang/definitions/test_json_schema.py

```diff
@@ -65,6 +65,9 @@ def test_get_definition_json_schema_with_model(self):
         "root": {
             "type": "string"
         },
+        "description": {
```

Comment:
I suspect this is an addition caused by adding the `description` field to the `schema` definition in the core spec. Although, IIRC, `schema` did have a `description` field at one point so I'm not sure I understand why it was needs to be re-added or how it got removed.

Reply:
Okay, so I'm not wrong in being confused about this! I'm not sure what happened either. But fixed now it seems.

Reply:
I believe JD added moved the old root field descriptions to the schema definitions which didn't previously have the description field defined.

Reply:
So, I added the old root field descriptions so that they got carried forward instead of lost. Was one of the changes I suggested on his PR. But guess I misinterpreted the relation for carrying them forward. That it was at the field level that I pulled them from and not higher in the model.